ui/src/common/SongDatagrid.js

```diff
@@ -49,12 +54,57 @@ const useStyles = makeStyles({
   },
 })
 
+const EditionRow = forwardRef(
+  ({ record, onClick, colSpan, contextAlwaysVisible }, ref) => {
+    const isDesktop = useMediaQuery((theme) => theme.breakpoints.up('md'))
+    const classes = useStyles({ isDesktop })
+    const translate = useTranslate()
+    const handlePlaySubset = (releaseDate) => () => {
+      onClick(releaseDate)
+    }
+
+    let editionTitle = []
+    if (record.releaseDate) {
+      editionTitle.push(translate('resources.album.fields.released'))
+      editionTitle.push(FormatFullDate(record.releaseDate))
+      if (record.catalogNum && isDesktop) {
+        editionTitle.push('· Cat #')
+        editionTitle.push(record.catalogNum)
+      }
+    }
+
+    return (
+      <TableRow
+        hover
+        ref={ref}
+        onClick={handlePlaySubset(record.releaseDate)}
```

Comment:
Why do we need the indirection? Can't we just do this?

```suggestion
        onClick={onClick}
```

Reply:
I don't know, I didn't investigate - this bit was already in the code before as `onClick={handlePlayDisc(record.discNumber)}`, I just copied and modified it to work the same way with releases

Reply:
Ah, ok. No worries then. I can take a look later.